bitmap-raziyehLogan/lib/bitmap-file-write.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var fs = require('fs');
+
+module.exports = function(outputfileName, buffer, callback){
```

Comment:
@loganlsr @Raziyehbazargan - the naming convention for your output file is a bit too lengthy - go with something shorter - `newFile`, `outputFile`, or something of that nature